### DISCUSSION TITLE
SyntaxError: invalid syntax

### DISCUSSION THREAD

**Discussion by Junosupafly** [crewAIInc/crewAI#1146] [Q&A]
when i try to run the code on VS i get this error

  File "<stdin>", line 1
    & C:/Users/pauly/AppData/Local/Programs/Python/Python312/python.exe "c:/Users/pauly/crewAI/import os.py"
    ^
SyntaxError: invalid syntax
>>>

Please help

**Comment by Biancamazzi** [comment]
you can use https://chat.openai.com/g/g-qqTuUWsBY-crewai-assistant to help you with that

**Comment by C-1pher** [comment]
it looks like you have a problem in the imports of your code
